### PR TITLE
Add the ability to override fonts (resolves issue #65 )

### PR DESCRIPTION
**Pull request by avicndugu**
This is an enhancement or feature.

## Summary

With this new feature, users can add new fonts, or update current fonts by just editing the style.yml file.

How it works
- Use the "fonts_urls" to add any font to the website.
- Use the "fonts" to match a font mixin with the corresponding font name.

## Context

This is related to GitHub issue #65

### CODE REVIEW

**Comment by raviriley** [comment]
Thanks for the great work @avicndugu!

File: _includes/head.html
Comment (raviriley) [inline _includes/head.html:1]: I believe removing the current font links will break anyone who doesn't have the `style.yml` changes in their version of the project. I would add a check for the existence of custom fonts in `style.yml`, else default to the original links.

**Review by raviriley — CHANGES_REQUESTED** [review]

File: _includes/head.html
Comment (avicndugu) [reply _includes/head.html:1]: Thanks for the review. I missed providing a fallback.

I have implemented the check for custom fonts to ensure everything works as before without the custom fonts.

**Review by raviriley — APPROVED** [review]

**Comment by raviriley** [comment]
Thanks for the quick turnaround @avicndugu !

**Pull request merged**